### DISCUSSION TITLE
Calculated averages kilometers

### DISCUSSION THREAD

**Discussion by ovahead** [mytourbook/mytourbook#1244] [General]
Once again I have something that really confuses me. Why is the movement time exactly as long as the total time in some cases, even though there are interrupted times and the recording time is also less than the total time? In this context, I would also like to know the  MyTourbook definition of total time, recording time, interrupted time, movement time and break time. What is the difference between interrupted time and break time?

I come to this because it seems to me that Mytourbook uses incorrect values for calculating the average kilometers. For example, while my Garmin shows me an average speed of 20.92 km/h, Mytourbook says it would only be 10.24

During my short shopping trip yesterday with more standstill than travel time, Mytourbook shows me 4.51 km/h and my Garmin says 19.41.
These are differences that no longer have anything to do with each other and to me it looks like something isn't working properly :(
I have to say that I have previously edited the original Garmin FIT file with FitFileRepaitTool but only for altitude information and not any time information.

![MT---Kurzer-Shopping-Trip](https://github.com/mytourbook/mytourbook/assets/112829463/f581e3c2-9bae-4c54-8e7b-716c1e05adfb)
![Garmin-Edge-830---Shopping-Trip](https://github.com/mytourbook/mytourbook/assets/112829463/598c4f8e-d08b-4948-9872-85ef737269b1)

**Comment by FJBDev** [comment]
> In this context, I would also like to know the MyTourbook definition of total time, recording time, interrupted time, movement time and break time. What is the difference between interrupted time and break time?

My PR [here](https://github.com/mytourbook/mytourbook/pull/222) explains the recorded time and paused times.

[Here](https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/preferences/general/pref-computed-values#break-time) you can find explanations of moving time and break times
